target/linux/ramips/dts/Telco-Electronics-X1.dts

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "Telco-Electronics-X1.dtsi"
```

Comment:
I'm curious: Is there another, but related product like the X1? Because if not please integrate the .dtsi into the .dts. Thanks.